### ISSUE TITLE
Definición para el stack de gemas a usar en el proyecto.

### ISSUE THREAD

**Issue by gbrlmrllo** [rubyespanol/Primer_Blog#1]
## Autenticación
* Devise

**Comment by roadev** [comment]
sass-rails
autoprefixer-rails

**Comment by Oxyrus** [comment]
Autenticación: Devise
Autorización: CanCanCan
Formularios: simple_form

**Comment by gbrlmrllo** [comment]
## Autenticacion y Autorización
* Integración para autenticación de usuarios con [Devise](https://github.com/plataformatec/devise)
* Integración para roles de usuarios con [Rolify](https://github.com/RolifyCommunity/rolify)
* Integración para autorizaciones con [CanCanCan](https://github.com/CanCanCommunity/cancancan)

## Paginación
* Inegración para el manejo de de paginación con [Kaminari](https://github.com/amatsuda/kaminari)

## Formularios
* Integración para helpers de formularios con [SimpleForm](https://github.com/RolifyCommunity/rolify)

## Busqueda
* Integración para búsquedas full-text con [ElasticSearch](https://github.com/elastic/elasticsearch-rails)

## Diseño frontend
* Integración con framework fronte-end para el administrativo con [Bootstrap](https://github.com/twbs/bootstrap-sass)

**Comment by roadev** [comment]
Me gusta la idea de kaminari para la paginación.

**Comment by Oxyrus** [comment]
Podemos utilizar FriendlyId para los slugs

**Comment by ssoulless** [comment]
Me gustaría sugerir Zurb Foundation 6 para el frontend, en mi opinión más potente que bootstrap y [esto](http://foundation.zurb.com/templates.html) nos ahorraría algo de tiempo

**Comment by Oxyrus** [comment]
Sí, http://foundation.zurb.com/templates-previews-sites-f6/blog.html se ve bastante decente

**Comment by roadev** [comment]
Hace mucho he tenido ganas de hacer algo en foundation :D @ssoulless

**Comment by ssoulless** [comment]
Hola creo que es muy importante que escribamos tests en nuestros PR, sugiero que al Stack se añada un framework de testing, yo propongo Rspec con Capybara, la discusión para estandarizar el testing y la herramienta de CI la podemos discutir en #24

**Comment by roadev** [comment]
@ssoulless he oído que es el framework más completo y grande para hacer testing...
Si ya lo has trabajado y como he visto, por aquí no somos unos duros en testing, creo que estaría bien para aprender con tu ayuda :)
De hecho no sé si en railskarate tengas aún algo de testing, podría salir contenido para los tutoriales de acá :D

**Comment by Frank004** [comment]
@roadev  muy cierto. a mi me paso ya que el cliente que tuve no me dejaba hacer nada de test todo era un deadline y tuve que aprender si hacer test so a la larga pasaba doble trabajo. cuando el cliente no sabe de tecnología  es bien difícil trabajar los test. :(  esto me va ayudar un mucho. me recuerdo que todo esto comenzó con un post de un compañero y despertó esta necesidad de unirnos.

**Comment by ssoulless** [comment]
Las gemas de testing: RSpec, Capybara y factory_girl ya están incluidas y configuradas para empezar a hacer testing. #39
